src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -263,6 +273,13 @@ void MediaManagerImpl::StopStreaming(
     int32_t application_key, protocol_handler::ServiceType service_type) {
   LOG4CXX_AUTO_TRACE(logger_);
 
+  end_time = std::chrono::system_clock::now();
+  std::chrono::duration<double> elapsed_seconds = end_time - start_time;
+  LOG4CXX_DEBUG(logger_,
```

Comment:
Oh boy I do not think we need more logs when streaming media.

Reply:
Agreed. Only using those for debugging. Removed in https://github.com/smartdevicelink/sdl_core/pull/2956/commits/687478c2d0acbd01664dcd0bc8d518d9900ce905